extensions/wikia/WikiFactory/Reporter/SpecialWikiFactoryReporter_body.php

```diff
@@ -122,7 +122,7 @@ function getCustomSettings()
 				#$cv_value = 'Error. Not an array?!?';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app